recipes/cryptography/meta.yaml

```diff
@@ -0,0 +1,81 @@
+{% set name = "cryptography" %}
+{% set version = "1.9" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 5518337022718029e367d982642f3e3523541e098ad671672a90b82474c84882
+
+build:
+  number: 0
+  script:
+    - export CFLAGS="-I${PREFIX}/include ${CFLAGS}"  # [unix]
+    - export LDFLAGS="-L${PREFIX}/lib ${LDFLAGS}"  # [unix]
+    # needed to link to openssl 1.0.*
+    - set CRYPTOGRAPHY_WINDOWS_LINK_LEGACY_OPENSSL=1  # [win]
+    - python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - toolchain
+    - python
+    - setuptools >=11.3
+    - openssl 1.0.*
+    - cffi >=1.7
+    - enum34  # [py2k]
+    - ipaddress  # [py2k]
+    - asn1crypto >=0.21.0
+    - six >=1.4.1
+    - idna >=2.1
```

Comment:
Yep that seems reasonable.

Side note: GitHub has moved this comment out of context. It was in the `build` section, but the issue raised has been subsequently addressed.